plugins/modules/notification/telegram.py

```diff
@@ -11,44 +11,76 @@
 DOCUMENTATION = '''
 
 module: telegram
-author: "Artem Feofanov (@tyouxa)"
+author:
+ - "Artem Feofanov (@tyouxa)"
+ - "Nikolai Lomov (@lomserman)"
 
 short_description: module for sending notifications via telegram
 
 description:
-    - Send notifications via telegram bot, to a verified group or user
+    - Send notifications via telegram bot, to a verified group or user.
+    - Also, you may try to use any other telegram bot api method, if you specify "api_method" argument.
```

Comment:
Add a change log entry for this change like - https://github.com/ansible-collections/community.general/tree/main/changelogs/fragments

Reply:
@Akasurde fragment added. Thx for review btw.